tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "es6",
+    "jsx": "react"
+  }
+}
```

Comment:
For tsconfig you can try this configuration:
```
{
  "compilerOptions": {
    "noImplicitAny": false,
    "module": "esnext",
    "sourceMap": true,
    "target": "es2015",
    "moduleResolution": "node",
    "removeComments": true,
    "lib": ["dom", "es2015"],
    "jsx": "react",
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "experimentalDecorators": true,
    "declaration": true,
    "declarationDir": "@types"
  },
  "exclude": [
    "node_modules",
    "**/*.spec.ts"
  ]
}
```

But test it. Probably can throw error in few places of your project